.github/workflows/tests.yml

```diff
@@ -8,27 +8,27 @@ jobs:
         python-version:
           - "3.8"
           - "3.9"
-        django-version:
-          - "3.2"
-          - "4.0"
-          - "4.1"
-          - "4.2"
+          - "3.10"
+          - "3.11"
+          - "3.12"
```

Comment:
I’ve moved the GitHub setup to only use a per-Python matrix, using tox to run the relevant environments per version. This is what I use on my ~30 open source projects and it works really well. It requires less maintenance and is actually faster than splitting per Django version unless you have a long test suite.